itou/users/models.py

```diff
@@ -406,6 +406,10 @@ def is_job_seeker(self):
     def is_prescriber(self):
         return self.kind == UserKind.PRESCRIBER
 
+    @property
+    def is_orienter(self):
+        return self.is_prescriber and not self.prescriberorganization_set.exists()
```

Comment:
J'ai un doute mais je suis pas sûr qu'il y ai besoin du `.exists()`, mais peut-être que Django optimise déjà en fonction du cas.

Reply:
Je vérifie. Merci !

Reply:
Le set existe dans les deux cas, donc le `.exists()` fait bien un différence

Reply:
Euh a priori un prescripteur d'une organisation non-habilitée est un orienteur cf 
https://github.com/betagouv/itou/blob/02a3f8bc9d5d601e0f6862bf569aeb7aa7e99024/itou/prescribers/models.py#L68-L94

Donc si on veut un `is_orienteur`, je verrais plutôt:
`self.is_prescriber and not self.is_prescriber_with_authorized_org `